loopout.go

```diff
@@ -75,7 +75,8 @@ type executeConfig struct {
 
 // loopOutInitResult contains information about a just-initiated loop out swap.
 type loopOutInitResult struct {
-	swap *loopOutSwap
+	swap          *loopOutSwap
+	serverMessage string
```

Comment:
idk about adding a struct for our struct rather than just returning `loopOutSwap, string, error`

Reply:
I had that first and changed it to this. It felt that the server message got more attention than it is worth, but totally subjective